apps/site/lib/site_web/templates/event/_calendar.html.eex

```diff
@@ -38,6 +38,17 @@
                   conn: @conn,
                   event: event,
                   class: if index >= 4, do: "open-left" %>
+
+                  <noscript>
+                    <%= link to: cms_static_page_path(@conn, event.path), class: "m-event-calendar__event m-event-calendar__event-title", title: event.title, target: "_blank" do %>
```

Comment:
Can we either add a warning that this will open in a new window ([see rationale and examples in this linked accessibility guidance](https://www.w3.org/TR/WCAG20-TECHS/G201.html)) or remove the `target: "_blank"` from these links? 

I have no preference for which option to take, but removing the attribute might make more sense since the event links in the JS-enabled popup don't open in a new window anyway.

Reply:
Sure thing, I'll remove it.